src/features/pilot_management/Print/MechPrint.vue

```diff
@@ -258,7 +258,7 @@
           <v-spacer />
           <span v-if="w.Uses">
             <v-icon
-              v-for="n in w.MaxUses + mech.Pilot.LimitedBonus"
+              v-for="n in w.getTotalUses(mech.Pilot.LimitedBonus)"
```

Comment:
Whenever we can, we should use the built-in `getTotalUses` over `+` operations when calculating Total Limited Uses.